web/cobrands/sass/_layout.scss

```diff
@@ -159,10 +174,21 @@ h1 {
 
 // .content Is the white box
 .content {
-  margin: 1em 0 0 0;
```

Comment:
Why have we lost the 1em top margin of global `.content`? e.g. the help page on Bromley is now squashed up against the heading when it wasn't before. And then in frontpage you're putting it back, so I don't think it should be removed here. Think this might have happened because of a change then a not identical change back in the fixup.

Reply:
@dracos I have included a commit to fix this.